packages/twenty-front/src/modules/page-layout/widgets/graph/utils/getSortedKeys.ts

```diff
@@ -1,18 +1,78 @@
+import { type FieldMetadataItemOption } from '@/object-metadata/types/FieldMetadataItem';
+import { type RawDimensionValue } from '@/page-layout/widgets/graph/types/RawDimensionValue';
+import { sortByManualOrder } from '@/page-layout/widgets/graph/utils/sortByManualOrder';
+import { sortBySelectOptionPosition } from '@/page-layout/widgets/graph/utils/sortBySelectOptionPosition';
+import { isDefined } from 'twenty-shared/utils';
 import { GraphOrderBy } from '~/generated/graphql';
 
 export const getSortedKeys = ({
```

Comment:
This should be renamed, it's too vague, if it shows up in a research with cmd+k it is hard to understand what it does.